File: sdkwork-backend-sdk-kotlin/src/main/kotlin/com/sdkwork/backend/PlusGameGrowthAccountRefreshForm.kt
package com.sdkwork.backend

data class PlusGameGrowthAccountRefreshForm(
    val userId: Int? = null,
    val accountType: String? = null,
    val gameType: String? = null,
    val gameModeCode: String? = null,
    val seasonKey: String? = null
)

File: sdkwork-backend-sdk-typescript/src/types/plus-game-growth-account-refresh-form.ts
/** Growth account refresh form */
export interface PlusGameGrowthAccountRefreshForm {
  /** User ID */
  userId: number;
  /** Growth account type */
  accountType: 'PLAYER_LEVEL' | 'GAME_LEVEL';
  /** Game type, required for game-level growth accounts */
  gameType?: 'INTERNATIONAL_CHESS' | 'CHINESE_CHESS' | 'GOMOKU' | 'JUNQI' | 'LANDLORD' | 'SHENG_JI' | 'TUO_LA_JI' | 'TEXAS_HOLDEM' | 'MAHJONG' | 'CUSTOM';
  /** Game mode code */
  gameModeCode?: string;
  /** Season key */
  seasonKey?: string;
}
